tests/src/Search/ExampleWebTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace tests\eLife\Search;
+
+/**
+ * @group web
+ */
+class ExampleWebTest extends ElasticTestCase
+{
+    /**
+     * @test
+     */
+    public function testCanRunCommand()
+    {
+        // Run command
+        $lines = $this->runCommand('hello');
+        // And test.
+        $this->assertEquals('Hello from the outside (of the global scope)', $lines[0]);
+        $this->assertEquals('This is working', $lines[1]);
+    }
+
+    /**
+     * @test
+     */
+    public function testElasticSearchIndex()
+    {
+        $this->addDocumentToElasticSearch('
```

Comment:
would produce better error messages to perform `json_encode()` over a PHP array, because a syntax error would be difficult to catch in the current state